### PR TITLE
FIO Address Details Scene themed.

### PR DESCRIPTION
**Pull request by andreyvEze**
Pixel 3
Снимок экрана 2020-12-03 в 18 43 20

### CODE REVIEW

File: src/modules/FioAddress/components/ConnectWallets.js [hunk 1]
```diff
@@ -200,22 +203,22 @@ class ConnectWallets extends React.Component<FioConnectWalletStateProps & OwnPro
...
   list: {
-    flex: 5,
-    backgroundColor: THEME.COLORS.WHITE
+    flex: 5
   },
   no_wallets_text: {
-    padding: scale(30),
-    fontSize: scale(22),
-    color: THEME.COLORS.GRAY_2,
+    padding: theme.rem(1.875),
```
Comment (paullinator) [inline src/modules/FioAddress/components/ConnectWallets.js:211]: these should be more evenly divisible rem values such as 1.25 or 1.5

File: src/modules/FioAddress/components/ConnectWallets.js [hunk 2]
```diff
@@ -2,17 +2,16 @@
...
 
 import { showError } from '../../../components/services/AirshipInstance'
+import { type Theme, type ThemeProps, cacheStyles, withTheme } from '../../../components/services/ThemeContext'
 import * as Constants from '../../../constants/indexConstants'
 import s from '../../../locales/strings.js'
-import { THEME } from '../../../theme/variables/airbitz.js'
 import { type RootState } from '../../../types/reduxTypes'
 import type { FioConnectionWalletItem } from '../../../types/types'
-import { scale } from '../../../util/scaling.js'
 import T from '../../UI/components/FormattedText/FormattedText.ui.js'
```
Comment (paullinator) [inline src/modules/FioAddress/components/ConnectWallets.js]: do not use T component. use EdgeText instead since it is themed

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/modules/FioAddress/components/ConnectWallets.js (hunk 1 above)
Comment (andreyvEze) [reply src/modules/FioAddress/components/ConnectWallets.js:211]: ok

File: src/modules/FioAddress/components/ConnectWallets.js (hunk 2 above)
Comment (andreyvEze) [reply src/modules/FioAddress/components/ConnectWallets.js]: yes, sure, missed that

Comment (andreyvEze) [reply src/modules/FioAddress/components/ConnectWallets.js]: @paullinator you know, it has already changed :)

**Comment by paullinator** [comment]
can you submit the screenshots for Google Pixel 3 and iPhone 11 for each of the PRs you have submitted

**Comment by andreyvEze** [comment]
ok, I will

**Pull request merged**